server/db/index.js

```diff
@@ -4,115 +4,134 @@ const db = require('./db')
       ImgUrl: '/images/sze_strawberries.jpg'
     }),
     Product.create({
-      name: 'Blueberries, 1 Pint', 
-      price: 1.99 , category: 'Fresh Produce', 
-      quantity: 10, 
-      descripton: 'These sweet, tangy and intensely blue berries contain antioxidants such as anthocyanin, as well as anti-inflammatories. It has sweet-tart taste and their unique blue color.', 
+      name: 'Blueberries, 1 Pint',
+      price: 1.99, category: 'Fresh Produce',
+      quantity: 10,
+      descripton: 'These sweet, tangy and intensely blue berries contain antioxidants such as anthocyanin, as well as anti-inflammatories. It has sweet-tart taste and their unique blue color.',
       ImgUrl: '/images/sze_blueberries.jpg'
     }),
     Product.create({
-      name: 'Bananas,(5 ct)', 
-      price: 0.99 , category: 'Fresh Produce', 
-      quantity: 10, 
-      descripton: 'Characterized by their bright yellow color and sweet taste, bananas are the ideal fruit for snacks and sports activities', 
+      name: 'Bananas,(5 ct)',
+      price: 0.99, category: 'Fresh Produce',
+      quantity: 10,
+      descripton: 'Characterized by their bright yellow color and sweet taste, bananas are the ideal fruit for snacks and sports activities',
       ImgUrl: '/images/sze_bananas.jpg'
-    }), 
+    }),
     Product.create({
-      name: 'Cucumber', 
-      price: 1.69, category: 'Fresh Produce', 
-      quantity: 10, 
-      descripton: 'Cucumber is a widely cultivated plant ', 
+      name: 'Cucumber',
+      price: 1.69, category: 'Fresh Produce',
+      quantity: 10,
+      descripton: 'Cucumber is a widely cultivated plant ',
       ImgUrl: '/images/sze_cucumber.jpg'
     }),
     Product.create({
-      name: 'Avocado, One Large', 
-      price: 0.99, category: 'Fresh Produce', 
-      quantity: 10, 
-      descripton: 'Avocado fruits have greenish or yellowish flesh with a buttery consistency and a rich, nutty flavour. ', 
+      name: 'Avocado, One Large',
+      price: 0.99, category: 'Fresh Produce',
+      quantity: 10,
+      descripton: 'Avocado fruits have greenish or yellowish flesh with a buttery consistency and a rich, nutty flavour. ',
       ImgUrl: '/images/sze_avocado.jpg'
     }),
     Product.create({
-      name: 'Organic Green Onion (Scallions), 1 Bunch', 
-      price: 0.99, category: 'Fresh Produce', 
-      quantity: 10, 
-      descripton: 'Organic Green Onion (Scallions), One Bunch. Both parts of the green onion are edible and can be used in cooking. Aid in Weight Loss. ', 
+      name: 'Organic Green Onion (Scallions), 1 Bunch',
+      price: 0.99, category: 'Fresh Produce',
+      quantity: 10,
+      descripton: 'Organic Green Onion (Scallions), One Bunch. Both parts of the green onion are edible and can be used in cooking. Aid in Weight Loss. ',
       ImgUrl: '/images/sze_onion.jpg'
```

Comment:
I right clicked and did FORMAT DOCUMENT, i guess its picking it up as a change, but I didn't touch the product stuff